config.yaml

```diff
@@ -14,3 +14,7 @@ options:
     description: enables auto creation of topic on the server
     type: boolean
     default: false
+  external-listener:
```

Comment:
What is the long term likelihood of this being set via config? 
Am I correct in assuming that down the line, this will be handled by some ingress (traefick/nginx/etc) charm that will handle this setting from within a lib?

If so, and we're assuming that this will be removed at some point, Rob's rename suggestion is good.

Reply:
I'm removing this config option completely. It cannot be used for this issue.